Source/Core/DolphinWX/GameListCtrl.cpp

```diff
@@ -237,6 +238,9 @@ static bool ShouldDisplayGameListItem(const GameListItem& item)
   if (!show_platform)
     return false;
 
+  if (item.GetPlatform() == DiscIO::Platform::WII_WAD && !IOS::ES::IsChannel(item.GetTitleID()))
+    return false;
```

Comment:
With your implementation, ISOFile.cpp is able to read from the game list cache instead of having to create volume objects, so we don't read anything from the WAD file itself.